src/seedsigner/views/screensaver.py

```diff
@@ -19,8 +19,10 @@
 # TODO: This early code is now outdated vis-a-vis Screen vs View distinctions
 class LogoScreen(BaseScreen):
     def __init__(self):
+        """Initialize the logo display screen with the SeedSigner logo image path, partner logos, and active state."""
         super().__init__()
-        self.logo = load_image("logo_black_240.png")
+        self.image_path = load_image("logo_black_240.png")  # Renamed for consistency
```

Comment:
`load_image` returns a `PIL.Image` object. This proposed rename confuses what is being operated on in later calls.